src/microbe_stage/particles/background/Current.tscn

```diff
@@ -0,0 +1,59 @@
+[gd_scene load_steps=9 format=2]
```

Comment:
I think there are also unused scenes in the particles and particles/background folders that should be moved to that experimental folder I mentioned before.